locales/en.json

```diff
@@ -71,28 +71,28 @@
 	"Header": "Header",
 	"Unordered List": "Unordered List",
 	"Ordered List": "Ordered List",
-	"Todo List": "Todo List",
+	"Todo List": "Checklist",
 	"Blockquote": "Blockquote",
 	"Bold font": "Bold font",
 	"Italics font": "Italics font",
 	"Strikethrough": "Strikethrough",
 	"Inserted text": "Inserted text",
-	"Marked text": "Marked text",
+	"Marked text": "Highlighted text",
 	"Link": "Link",
 	"Image": "Image",
 	"Code": "Code",
 	"Externals": "Externals",
-	"This is a alert area.": "This is a alert area.",
-	"Revert": "Revert",
+	"This is a alert area.": "This is an alert area.",
+	"Revert": "Undo",
 	"Import from clipboard": "Import from clipboard",
 	"Paste your markdown or webpage here...": "Paste your markdown or webpage here...",
 	"Clear": "Clear",
 	"This note is locked": "This note is locked",
-	"Sorry, only owner can edit this note.": "Sorry, only owner can edit this note.",
+	"Sorry, only owner can edit this note.": "Sorry, only the owner can edit this note.",
```

Comment:
Can the "the" be ignored?
I would prefer shorter and clear wording in most of the UI text.